compiler/rustc_codegen_llvm/src/debuginfo/metadata/enums/mod.rs

```diff
@@ -265,6 +265,8 @@ fn build_enum_variant_struct_type_di_node<'ll, 'tcx>(
                 variant_index,
             ),
             variant_def.name.as_str(),
+            unknown_file_metadata(cx),
+            UNKNOWN_LINE_NUMBER,
```

Comment:
If we can manage it, it would be nice to use the enum variant declaration line as mentioned above.